wiring_color_code_printer.py

```diff
@@ -0,0 +1,13 @@
+import wiring_color_code_handler
+
+def print_color_code_reference():
+    row_format = "{:>15}" * (3)
+    table_headers = ["Pair number", "Major Color", "Minor Color"]
+    print(row_format.format("","25-Pair Color Code",""))
+    print(row_format.format(*table_headers))
+    for i in range(1, 25):
```

Comment:
The background behind the 25 is not apparent. A maintainer is likely to forget updating this number when a new color is added. How could we address this?

Reply:
Yes, that's a very likely scenario. I can see this in hindsight now. Would declaring the colors and the number of color pairs in a Constant.py file be an appropriate resolution?